lizrd/grid/infrastructure.py

```diff
@@ -331,7 +331,7 @@ def get_subprocess_args(
     ):
         if n_consecutive != 1:
             raise Exception(
-                "You are trying to on the repeater mode on a cluster that do not not support that option."
+                "You are trying to on the repeater mode (aka. checkpoint manager) on a cluster that do not not support that option."
```

Comment:
"You are trying to turn on the repeater mode (aka. checkpoint manager) on a cluster that does not support that option." ?

Reply:
changed

Reply:
There is still a grammatical mistake (do -> does).